react-native/android/build.gradle

```diff
@@ -27,8 +27,8 @@ import org.apache.tools.ant.filters.ReplaceTokens
 // We download various C++ open-source dependencies into downloads.
 // We then copy both the downloaded code and our custom makefiles and headers into third-party-ndk.
 // After that we build native code from src/main/jni with module path pointing at third-party-ndk.
-ext.coreVersion = '0.96.2'
 
+ext.coreVersion = "$System.env.REALM_CORE_VERSION"
```

Comment:
Good idea, but we support calling gradle directly, so I would switch this to add an `npm run` script to `package.json` (like `get-core-version`, which calls `scripts/download-core.sh --version`) and run that just like the line below that sets `currentVersion`